bindings/python/test/test_pandas.py

```diff
@@ -376,7 +375,7 @@ def assert_in_idx(self, table, col_name):
         int: ["int64", "float64"],
         float: "float64",
         datetime.datetime: "datetime64[ns]",
-        ObjectId: "object",
+        ObjectId: "bson_PandasBSONObjectId",
```

Comment:
What do you think about renaming these types to just `PandasObjectId` or similar so that the names become `bson_PandasObjectId` rather than repeating BSON twice?

Reply:
Done